src/components/Modal/index.style.tsx

```diff
@@ -1,22 +1,77 @@
-import { cva } from "class-variance-authority";
-
-export const ModalOverlayStyles = cva([
-  "fixed inset-0 flex items-center justify-center bg-black bg-opacity-25 transition-opacity duration-300 ease-in-out z-10 ",
-]);
-
-export const ModalBodyStyles = cva(
-  "bg-white dark:bg-modalColor overflow-hidden rounded-md shadow-lg transition-transform duration-300 ease-in-out flex flex-col",
-  {
-    variants: {
-      size: {
-        sm: "w-full max-w-sm m-4 max-h-[90vh]",
-        md: "w-full max-w-md m-4 max-h-[90vh]",
-        lg: "w-full max-w-lg m-4 max-h-[90vh]",
-        full: "w-full h-full m-0 max-h-screen rounded-none",
-      },
-    },
-    defaultVariants: {
-      size: "md",
-    },
+/** @jsxImportSource @emotion/react */
+import { css } from "@emotion/react";
+
+export const modalOverlayStyles = css`
+  position: fixed;
+  inset: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: rgba(0, 0, 0, 0.25);
+  transition: opacity 300ms ease-in-out;
+  z-index: 10;
```

Comment:
@Kashyap2701 Is there a reason why we use `10` as z-index? I think `1` would be enough as z-index. Thoughts?

Reply:
Using a higher value like 10 could safeguard the modal to overlay all other content, especially if the application already has elements with z-index values around 1 to 9.

It's a common practice to reserve lower z-index values for minor overlays, while higher values are used for significant UI elements like modals, tooltips, and notifications.